docs/content/doc/help/seek-help.en-us.md

```diff
@@ -20,10 +20,25 @@ menu:
 
 **NOTE:** When asking for support, it may be a good idea to have the following available so that the person helping has all the info they need:
 
-1. Your `app.ini` (with any sensitive data scrubbed as necessary)
-2. The `gitea.log` (and any other appropriate log files for the situation)
-    * e.g. If the error is related to the database, the `xorm.log` might be helpful
-3. Any error messages you are seeing
+1. Your `app.ini` (with any sensitive data scrubbed as necessary).
+2. The Gitea logs, and any other appropriate log files for the situation.
+    * The logs may be outputted to console by default. If you need to collect logs from files, 
```

Comment:
It may be a default? That doesn't sound like a default setting 😄.

Reply:
It depends on how the `app.ini` is generated.

Reply:
I think we can assume it's logged to the console one way or another. Most would hopefully be running in docker or via systemd in which they use existing configurations which should have it enabled.

Reply:
Sorry but I didn't get the point of your comment. What should be done?

Reply:
It sounds confusing to me, something like:
```suggestion
    * The logs are likely to be outputted to console. If you need to collect logs from files, 
```
Sounds quite more clear IMO.

Reply:
It is `MODE=console` by default , in most cases. But some setup steps can overwrite it.

![image](https://user-images.githubusercontent.com/2114189/175205703-3d2dcb37-506b-4750-ad6d-845004e5934b.png)

Reply:
I'm going to assume general case and those that have overwritten it are probably knowledgeable enough to understand that.

Reply:
Nope, they may not have such knowledge, for example, overwritten by docker environment, or some configurations generated by helmchart, or downstream packagers.